src/types.tsx

```diff
@@ -0,0 +1,17 @@
+export interface Todo {
+  id: number,
+  title: string,
+  completed: boolean,
+  userId: number,
+}
```

Comment:
You can add user to Todo type as optional property

```suggestion
export interface Todo {
  id: number,
  title: string,
  completed: boolean,
  userId: number,
  user?: User | null
}
```